### DISCUSSION TITLE
Bluetooth vs. USB

### DISCUSSION THREAD

**Discussion by christophhoevel** [FreeSpacenav/spacenavd#113] [Questions and support] [answered]
Hello community, is spacenavd working on Linux Ubuntu 24.04 with a bluetooth SpaceMouse or USB only? Many thanks, Christoph

**Comment by jtsiomb** [comment]
I think it should work. As long as the kernel talks to the device and makes it available through evdev, spacenavd should not be able to even know the difference.

Let us know if you try it.

**Comment by Wokoeng** [comment, marked as answer]
Hi, I got the Bluetooth Spacemouse working on PopOS 22.04 using the bulit-in Bluetooth modem on my Lenovo P50 laptop, no universal receiver dongle needed. Works flawlessly with Freecad 1.0. I didn't use the ancient spacenavd that is in the repos, but used the latest version from https://launchpad.net/ubuntu/+source/spacenavd for Plucky Puffin, which worked also on my Jammy- based PopOS. I compiled the Gui myself, which also works perfectly.  Cheers, Wolfgang

**Comment by christophhoevel** [comment]
Hello Wolfgang,

thanks for letting me know. 

I got it working now on my HP workstation. I am using an Asus bluetooth dongle and it is working perfectly with the bluetooth SpaceMouse under Ubuntu 22.04 and my bespoke software based on Qt 6.8.

I used the drivers and daemon from the spacenav repo, but will give the Ubuntu drivers a try.  I didnt spot them, so thanks for the hint.

Cheers
Christoph
